be/src/runtime/load_channel_mgr.cpp

```diff
@@ -134,6 +134,21 @@ Status LoadChannelMgr::open(const PTabletWriterOpenRequest& params) {
     return Status::OK();
 }
 
+Status LoadChannelMgr::open_partition(const PartitionOpenRequest& params) {
+    UniqueId load_id(params.id());
+    std::shared_ptr<LoadChannel> channel;
+    auto it = _load_channels.find(load_id);
+    if (it != _load_channels.end()) {
+        channel = it->second;
+    } else {
+        std::stringstream ss;
+        ss << "unknown load id, load id=" << load_id;
```

Comment:
No need to use stringstream here, can directly format in `Status::InternalError`.